setup.py

```diff
@@ -57,7 +57,7 @@ def local_file(name):
 extras[":python_version == '2.7'"] = ['enum34']
 extras[":python_version == '3.3'"] = ['enum34']
 
-install_requires = []
+install_requires = ['attrs', 'coverage']
```

Comment:
We don’t want _any_ pins on these libraries? Not even, say, to the current major version?

Reply:
attrs doesn't do major versions (it's on calver), and we're using a really really stable amount of its API. So, maybe? Probably worth doing. We almost certainly want to be quite precise about coverage versions though, yeah.